### PR TITLE
Release 22.12 to master

### PR DESCRIPTION
**Pull request by fmontes**
### Proposed Changes
* change 1
* change 2

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Review by dsilvam — APPROVED** [review]

File: docker/docker-compose-examples/push-publish/docker-compose-receiver.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ services:
       - opensearch-net_receiver
 
   dotcms-receiver:
-    image: dotcms/dotcms:latest
+    image: dotcms/dotcms:22.11_16e848e4_SNAPSHOT
```
Comment (dsilvam) [inline docker/docker-compose-examples/push-publish/docker-compose-receiver.yml:36]: undo

File: docker/docker-compose-examples/push-publish/docker-compose-sender.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ services:
       - opensearch-net_sender
 
   dotcms-sender:
-    image: dotcms/dotcms:latest
+    image: dotcms/dotcms:22.11_16e848e4_SNAPSHOT
```
Comment (dsilvam) [inline docker/docker-compose-examples/push-publish/docker-compose-sender.yml:36]: undo

File: docker/docker-compose-examples/push-publish/docker-compose-receiver.yml (hunk 1 above)
Comment (jcastro-dotcms) [reply docker/docker-compose-examples/push-publish/docker-compose-receiver.yml]: Done!

File: docker/docker-compose-examples/push-publish/docker-compose-sender.yml (hunk 2 above)
Comment (jcastro-dotcms) [reply docker/docker-compose-examples/push-publish/docker-compose-sender.yml]: Done!

**Pull request merged**